src/api/courseItemOperations.tsx

```diff
@@ -0,0 +1,42 @@
+import { CourseItem } from 'api';
+import { AddItemData } from 'components/AddItemModal';
+
+const handleErrors = (response: Response) => {
+	if (!response.ok) {
+		throw Error('fetch unsuccessful');
+	}
+	return response;
+};
+
+// create
+export const fetchPostCourseItem = async (data: AddItemData) =>
+	fetch('/courseItem/', {
+		method: 'POST',
+		headers: { 'Content-Type': 'application/json', userID: 'user1' },
+		body: JSON.stringify(data),
+	}).then(handleErrors);
+
+// read
+export const fetchGetCourseItem = async (id: string) =>
+	fetch(`/courseItem/${id}`, {
+		headers: { userID: 'user1' }, // REMINDER: REMOVE HADRCODED USER ID
```

Comment:
Are we removing the hard coded user id's later?

Reply:
I think so, unless we know how we're going to be getting the userid from the user already - although either way it might be nice to have it as its own ticket to keep the scope of the rest of these tickets down